database.rb

```diff
@@ -3,9 +3,7 @@
 def database_config
   case ENV['RACK_ENV']
   when 'production'
-    ActiveRecord::ConnectionAdapters::ConnectionSpecification::ConnectionUrlResolver.new(
-      ENV['DATABASE_URL']
-    ).to_hash
+    ENV['DATABASE_URL']
```

Comment:
This is great. Have you tested it works in production?

```
heroku run --app westconnex-m4-east-air-quality -e bundle exec irb
```

You could do that with irb, and running `AqmRecord.table_exists?` to test it.

Reply:
I've just tested it and it works ❤️ great refactor @henare ✨

Reply:
>Have you tested it works in production?

I didn't but I did test locally with:

    RACK_ENV=production DATABASE_URL="postgres://westconnex_m4east_aqm:password@localhost/westconnex_m4east_aqm_development" bundle exec dotenv ./scraper.rb

>You could do that with irb...

How does that work? Wouldn't it have been running the old code and therefore I couldn't test the change? :thinking: 

---

Thanks for the review and merge :trophy:

Reply:
>How does that work?

Oh I think I know what you mean now - I could copypasta my code into `irb` running on Heroku to see if that bit of the change worked. Is that what you were thinking?